msteams-platform/m365-apps/using-teams-client-sdk-preview.md

```diff
@@ -0,0 +1,365 @@
+---
+title: Microsoft Teams JavaScript client SDK v2 Preview
+description: Understand the changes coming with Microsoft Teams JavaScript client SDK v2 Preview
+ms.date: 11/15/2021
+ms.topic: conceptual
+---
+# Microsoft Teams JavaScript client SDK v2 Preview
+
+With the Microsoft Teams JavaScript client SDK v2 Preview, the [existing Teams SDK](/javascript/api/@microsoft/teams-js/) (`@microsoft/teams-js`, or simply `TeamsJS`) has been refactored to enable Teams developers the ability to [extend Teams apps to run in Outlook and Office](overview.md). From a functional perspective, the TeamsJS SDK v2 Preview (`@microsoft/teams-js@2.0.0-beta`) is a superset of the current TeamsJS SDK: it supports existing Teams app functionality while adding the ability to host Teams apps in Outlook and Office.
```

Comment:
We should either update the version to be `@microsoft/teams-js@2.0.0-beta.1` or use `@microsoft/teams-js@next` since 2.0.0-beta isn't a valid version and we have deprecated 2.0.0-beta.0.